requirements.txt

```diff
@@ -1,3 +1,3 @@
 django-netjsongraph>=0.5.0,<0.6.0
-openwisp-users>=0.1.12,<0.2
-openwisp-utils>=0.3.2,<0.4.0
+openwisp-users>=0.1.12,<0.3
+openwisp-utils>=0.3.2,<0.5.0
```

Comment:
Can you set the minimum to be 0.4 please? :smile:
Testing the rest.

Reply:
Done :smile: